x-pack/plugins/xpack_main/server/lib/setup_xpack_main.js

```diff
@@ -32,12 +32,6 @@ export function setupXPackMain(server) {
     }
   };
 
-  // trigger an xpack info refresh whenever the elasticsearch plugin status changes
-  server.plugins.elasticsearch.status.on('change', async () => {
-    await info.refreshNow();
```

Comment:
Pretty sure one of the reasons this being done is in case you change you license and then restart your cluster, not that it's a good way to cover that scenario. License info comes back in some sort of signature with each request now, right? Is that why we don't need this?

Reply:
I can add it back  in if we want, I removed it because it overlaps with https://github.com/elastic/kibana/pull/33584/files#diff-577f99d7103576efa3d57d5e26ec3fb5R138 now.  It may fire a little quicker if we leave it.

Reply:
I'm not picky either way, just want to make sure that this wasn't the only mechanism we had to refresh our cache when the license is updated in es.